beginners-luck.cabal

```diff
@@ -2,18 +2,18 @@
 --
 -- see: https://github.com/sol/hpack
 
-name:           beginners-luck
-version:        0.1.0.0
-category:       Web
-homepage:       https://github.com/githubuser/beginners-luck#readme
-bug-reports:    https://github.com/githubuser/beginners-luck/issues
-author:         Author name here
-maintainer:     example@example.com
-copyright:      2017 Author name here
-license:        BSD3
-license-file:   LICENSE
-build-type:     Simple
-cabal-version:  >= 1.10
+name:                beginners-luck
```

Comment:
With hpack, you should git rm the cabal file and let the file get ignored by the .gitignore declaration.